### PR TITLE
#javascript-javascript1-week4

### PR DESCRIPTION
**Pull request by sunilkumarmahajan**
Javascript1 - week4 home work ready for review

### CODE REVIEW

File: javascript/javascript1/week4/voiceassistant.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    else if (command.includes("What is my name")) 
+    {
+        return `Your name is ${inputname}` ;
+    }
+    else if (command.includes("Add fishing"))
+    {
+       todo.push(command);
+       return `fishing added to your todo` ;
+    }
+    else if (command.includes("Add singing"))
```
Comment (zuhairtaha) [inline javascript/javascript1/week4/voiceassistant.js:20]: Try to make a more dynamic solution to be able to get todo time whatever is it.
hint
```js
    const index1 = _command.indexOf('add ');
    const index2 = _command.indexOf(' to my todo');
    if (index1 !== -1 && index2 !== -1) {
      const todo = _command.substring(index1 + 'add '.length, index2);
}
```
Do the same for removing

**Review by zuhairtaha — CHANGES_REQUESTED** [review]
Good job.

File: javascript/javascript1/week4/voiceassistant.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            }
+        }
+    }
+    else if(command.includes("on my todo"))
+    {
+        return(todo);
+    }
+    else if (command.includes("What day is it today"))
+    {
+        const today = new Date().toDateString().split(" ");
```
[the remark this answers is not in the document]
Comment (sunilkumarmahajan) [reply javascript/javascript1/week4/voiceassistant.js]: done

File: javascript/javascript1/week4/voiceassistant.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        return(todo);
+    }
+    else if (command.includes("What day is it today"))
+    {
+        const today = new Date().toDateString().split(" "); 
+        return `${today[2]}. of ${today[1]} ${today[3]}`;
+    }
+    else if (command.includes("What is"))
+    {
+       const matharray = command.split(" ");
```
[the remark this answers is not in the document]
Comment (sunilkumarmahajan) [reply javascript/javascript1/week4/voiceassistant.js]: yes, I made the changes

**Pull request merged**